packages/docusaurus/lib/server/load/routes.js

```diff
@@ -34,7 +34,8 @@ async function loadRoutes(pluginsRouteConfigs) {
     const importStr = isObj ? target.path : target;
     const queryStr = target.query ? `?${stringify(target.query)}` : '';
     const chunkName = generateChunkName(name || importStr, prefix);
-    return `() => import(/* webpackChunkName: '${chunkName}' */ '${importStr}${queryStr}')`;
+    const finalStr = JSON.stringify(importStr + queryStr);
```

Comment:
What does `JSON.stringify` do here? Help escape another time?

Reply:
Yes. Otherwise it will be `C:\User` instead of `C:\\User`